apps/antalmanac/src/components/Calendar/Toolbar/CustomEventDialog/CustomEventDialog.tsx

```diff
@@ -156,6 +156,7 @@ class CustomEventDialog extends PureComponent<CustomEventDialogProps, CustomEven
                             variant="outlined"
                             size="small"
                             startIcon={<Add fontSize="small" />}
+                            disabled={AppStore.getSkeletonMode()}
```

Comment:
Creating or Editing while in skeleton mode should not be allowed IMO. Seems like it would be a damn easy place to create bugs and problems.